test/functional/mempool_package_onemore.py

```diff
@@ -48,10 +48,17 @@ def run_test(self):
         vout = utxo[0]['vout']
         value = utxo[0]['amount']
 
+        assert_equal(len(self.nodes[0].getrawmempool(True)), 0)
+
         fee = Decimal("0.0002")
         # MAX_ANCESTORS transactions off a confirmed tx should be fine
         chain = []
-        for _ in range(4):
+        # First tx has 3 outputs
```

Comment:
verified, this is correct, first tx has 3 outputs and the next three txs have 2 outputs

Reply:
heh I supposed this should just be an assert